print/components/scheduleEntry/contentNode/Checklist.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <content-node-content :content-node="contentNode" :icon-path="mdiClipboardListOutline">
+    <generic-error-message v-if="error" :error="error" />
+
+    <div v-for="entry in checklistsWithItems" :key="entry.checklist.id" class="tw-mb-2">
+      <div class="tw-font-bold">{{ entry.checklist.name }}</div>
+      <ul class="tw-list-disc tw-list-inside">
+        <li v-for="item in entry.items" :key="item.id">
+          <span class="tw-mr-1">{{ item.number }}</span> {{ item.text }}
```

Comment:
It would be good if the text would only wrap to the start of the text.
and maybe we don't need the list style (and just use the numbering, without the dot at the start)

![image](https://github.com/user-attachments/assets/26d6b5b9-7b16-4c1d-af68-712aae68e969)

Reply:
https://github.com/ecamp/ecamp3/pull/6031/commits/b2a175af69f097ca5e3b94081fe1a64635d6adb3

New visualization:
![image](https://github.com/user-attachments/assets/51014983-c50e-465f-a7e2-eee934ec9102)